### DISCUSSION TITLE
Flowcharting / Diagramming

### DISCUSSION THREAD

**Discussion by jwfuller** [nsidc/tech-radar#33] [Tools]
Lucid Charts is popular. It is Web-based and allows Co-editing (similar to Google Docs); however it is 
Proprietary / commercial (does not support open science) and does not allow us to version control it with code.

May consider

- [PlantUML](https://plantuml.com/)
- [Mermaid](https://mermaid.js.org/#/)

**Comment by MattF-NSIDC** [comment]
GitHub [supports Mermaid in Markdown](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/)

**Comment by trey-stafford** [comment]
I would agree that we should strive to use a non-commercial, open format for diagramming!

That said, it should be noted that NSIDC does have an org-level LucidChart account and many projects have used it for creating software diagrams (e.g., sea ice index, passive microwave, icebridge, etc.)
